modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/cmd/GetIdentityLinksForTaskCmd.java

```diff
@@ -43,6 +45,10 @@ public GetIdentityLinksForTaskCmd(String taskId) {
         CmmnEngineConfiguration cmmnEngineConfiguration = CommandContextUtil.getCmmnEngineConfiguration(commandContext);
         TaskEntity task = cmmnEngineConfiguration.getTaskServiceConfiguration().getTaskService().getTask(taskId);
 
+        if (task == null) {
+            throw new FlowableObjectNotFoundException("task " + taskId + " doesn't exist", Task.class);
+        }
+
```

Comment:
replaces the removed task existence check from all the endpoints retrieving task identity links, which was performed previously with `Task task = getTaskFromRequest(taskId);`

Reply:
also done again on the endpoint